storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst

```diff
@@ -0,0 +1,56 @@
+.. _rn.8-0-8:
+
+================================================================================
+|pxb.name| |release|
+================================================================================
+
+|percona| is glad to announce the release of |pxb.name| |release| on |date|.
+Downloads are available from our `download site
+<https://www.percona.com/downloads/Percona-XtraBackup-LATEST/>`_ and from
+:ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+|pxb.name| enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot
+tolerate long periods of downtime. Offered free as an open source solution, it
+drives down backup costs while providing unique features for |MySQL| backups.
+
+All |percona| software is open-source and free.
+
+New Features and Improvements
+================================================================================
+
+- Support log archiving feature in PXB 8.0. More information in
+  :jirabug:`PXB-1912` and in the `Redo Log section of MySQL documentation
+  <https://dev.mysql.com/doc/refman/8.0/en/innodb-redo-log.html>`_
+- Two options (`–-backup-lock-timeout` and `-–backup-lock-retry-count`) were
+  added to enable the configuring of the timeout for acquiring metadata locks in
+  ``FLUSH TABLES WITH READ LOCK``, ``LOCK TABLE FOR BACKUP``, and ``LOCK BINLOG
+  FOR BACKUP`` statements. More information in :jirabug:`PXB-1914`
+
```

Comment:
we should mention https://jira.percona.com/browse/PXB-1915 as new feature

Reply:
Added